model/bin/ww3_from_ftp.sh

```diff
@@ -10,19 +10,47 @@ curr_dir=`pwd`
 # Set WW3 code version
 ww3ver=v7.12.6
 
+interactive='n'
+keep='n'
+if [ $# -eq 1 ] ; then
+  if [ "$1" = "-i" ] ; then
+    interactive='y'
+  elif [ "$1" = "-k" ] ; then
+    keep='y'
+  else
+    echo '[ERROR] input argument not recognized'
+    echo '        use -i for interactive mode'
+    echo '        use -k to keep tar files'
+    exit 1
+  fi
+elif [ $# -gt 1 ] ; then
+  echo '[ERROR] only one input argument accepted'
+  echo '        use -i for interactive mode'
+  echo '        use -k to keep tar files'
+  exit 1
+fi
+
+model_dir=$(cd $(dirname $(dirname $0)) > /dev/null && pwd -P)
+echo $model_dir
+
 #Get top level directory of ww3 from user: 
 echo -e "\n\n This script will download data from the ftp for WAVEWATCH III "
 echo -e "Enter the relative path to the main/top level directory, this would "
 echo -e "be '../../' if in the model/bin directory or '.' if already in the "
 echo -e "top/main directory:"
```

Comment:
If we're giving the path to the model directory, we should change this description or is this still the same?

Reply:
done